### ISSUE TITLE
Det: Report that Screen Reader Can't Access Site

### ISSUE THREAD

**Issue by iph-97** [CityOfDetroit/bloom#1637]
Safiya in Detroit just passed along this report from a member of the City Office of Disability Affairs that a blind user said the site was inaccessible for a screen reader. Can we quickly confirm that screen readers are able to access the site and which screen reading programs we know work on the site? Email sent from the ODA staffer copied below. I told Safiya that once we confirm this, we would likely want to understand more details about the user's device and browser, screen reader, and specific issues with access to best address their concerns. This is a high priority item. 

"I hope you are doing well. I received feedback from a member of the blind community that the housing navigator website is not accessible to people who use screen readers. She also said the site had crashed before she was able to use it. I am happy to connect you to her if you would like to hear more from her, but I wanted to share the feedback with you to pass along to whoever worked on the site. Thanks!"

**Comment by sarahlazarich** [comment]
closing due to lack of response from stakeholder - will repoen if they respond
